src/Workspaces/Core/Portable/SymbolKey/SymbolKey.ErrorTypeSymbolKey.cs

```diff
@@ -13,9 +15,26 @@ private static class ErrorTypeSymbolKey
             public static void Create(INamedTypeSymbol symbol, SymbolKeyWriter visitor)
             {
                 visitor.WriteString(symbol.Name);
-                visitor.WriteSymbolKey(symbol.ContainingSymbol as INamespaceOrTypeSymbol);
-                visitor.WriteInteger(symbol.Arity);
+                switch (symbol.ContainingSymbol)
+                {
+                    case INamedTypeSymbol parentType:
+                        visitor.WriteInteger(0);
```

Comment:
Constants for 0/1/2 or do we have other places that don't do it?

Reply:
for this basic serialization/deserialization we're ok doing this for the simple writing/reading cases.  if it gets more complex we can consider named stuff.